filebeat/fileset/generate_fields_yml.go

```diff
@@ -0,0 +1,323 @@
+package fileset
```

Comment:
I would not mix the fileset package with the generator code. WDYT about `filebeat/generate/` package instead?